zlp-scheduler/db/migrate/20201123044850_add_activate_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddActivateToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :activate, :boolean, default: true
```

Comment:
Is there a reason why this is defaulting to true? It seems to me like the user should default to not being activated.

Reply:
That's a good point. The reason is that I don't want to have fewer changes.

There are a lot of tests calling `@user.save()` to create fake users.
And `@user.save()` is called in production when

1. Admin create another admin
2.  Admin upload student excel

If the default value is `true`, I only need to modify 2 locations for the production code.
If the default value is `false`, I need to update all the tests, otherwise, it may cause some problems.

 I think both `true` or `false` work for me, but I prefer to have fewer changes. What do you think?

Reply:
I think that's fine so long as the locations in the production code and updated correctly.